### PR TITLE
lens: Add version 3.6.8

### PR DESCRIPTION
**Pull request by VoX**

### CODE REVIEW

File: bucket/lens.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+                "script": [
+                    "Get-ChildItem \"$dir\" -Exclude app-32.7z | Remove-Item -Recurse",
+                    "Expand-7zipArchive \"$dir\\app-32.7z\" -Removal"
+                ]
+            }
+        }
+    },
+    "checkver": "github",
+    "autoupdate": {
+        "url": "https://github.com/lensapp/lens/releases/download/v$version/Lens-Setup-$version.exe#/dl.7z"
```
Comment (Ash258) [inline bucket/lens.json:36]:
```suggestion
    "description": "IDE for Kubernetes clusters",
    "homepage": "https://k8slens.dev",
    "license": "MIT",
    "url": "https://github.com/lensapp/lens/releases/download/v3.6.8/Lens-Setup-3.6.8.exe#/dl.7z",
    "hash": "sha512:3cd181d8a26f2a7f53f7571a6e7078454019d889c466362dfefa810087dd557038c403bee9deae8db6a24f0ed2c976d315639c1f923370086758464ecaa48884",
    "architecture": {
        "64bit": {
            "pre_install": [
                "Expand-7zipArchive \"$dir\\`$PLUGINSDIR\\app-64.7z\" \"$dir\"",
                "Remove-Item \"$dir\\`$*\", \"$dir\\Uninstall*\" -Recurse"
            ]
        },
        "32bit": {
            "pre_install": [
                "Expand-7zipArchive \"$dir\\`$PLUGINSDIR\\app-32.7z\" \"$dir\"",
                "Remove-Item \"$dir\\`$*\", \"$dir\\Uninstall*\" -Recurse"
            ]
        }
    },
    "shortcuts": [
        [
            "Lens.exe",
            "Lens"
        ]
    ],
    "checkver": {
        "github": "https://github.com/lensapp/lens"
    },
    "autoupdate": {
        "url": "https://github.com/lensapp/lens/releases/download/v$version/Lens-Setup-$version.exe#/dl.7z",
        "hash": {
            "url": "$baseurl/latest.yml",
            "regex": "sha512:\\s+$base64"
```

**Review by Ash258 — CHANGES_REQUESTED** [review]

File: bucket/lens.json (hunk 1 above)
Comment (VoX) [reply bucket/lens.json:36]: I tested the autoupdate and it seemed to work fine how I had it, but this seems like it should work as well.

Comment (VoX) [reply bucket/lens.json]: Oh I see you updated the homepage, tyvm

File: bucket/lens.json [hunk 2]
```diff
@@ -0,0 +1,38 @@
+{
+    "version": "3.6.8",
+    "description": "IDE for Kubernetes clusters",
+    "homepage": "https://k8slens.dev",
+    "license": "MIT",
```
Comment (Ash258) [inline bucket/lens.json:5]: Final product license is not MIT

A

```suggestion
    "license": {
        "identifier": "Proprietary",
        "url": "https://github.com/lensapp/lensapp.github.io/blob/master/licenses/eula.md"
    },
```

**Review by Ash258 — CHANGES_REQUESTED** [review]

**Review by Ash258 — APPROVED** [review]

**Pull request merged**